advisor/src/main/kotlin/Advisor.kt

```diff
@@ -65,12 +65,16 @@ class Advisor(
             return ortResult
         }
 
+        val packages = ortResult.getPackages(skipExcluded).map { it.pkg }
+        if (packages.isEmpty()) {
+            log.info { "Skipping advisor run, as there are no packages to advise." }
+            return ortResult
```

Comment:
I would prefer if it would still add an `AdvisorRun` to show that the advisor tool was executed. Otherwise it might give the false impression in reports that no advisor was executed.

Reply:
Good point. I changed the implementation to produce an empty result if there are no packages.